### DISCUSSION TITLE
How is the `ir.visualizer` option used?

### DISCUSSION THREAD

**Discussion by dee-see** [jruby/jruby#6739] [Q&A] [answered]
Hello team! I know of the `-Xir.print=true` option to view IR, but I see there's also and `ir.visualizer` option but I don't really understand how to use it as when I set it from the command line nothing seems to happen. Do you have any insight into that?

Thanks

**Comment by enebo** [comment, marked as answer]
This is a remnant of an old google summer of code project where all ir
passes were emitted to an external app for viewing. I doubt it works now.

-Tom
